dialogue-core/src/main/java/com/palantir/dialogue/core/DialogueChannel.java

```diff
@@ -165,6 +165,9 @@ public DialogueChannel build() {
                         new TraceEnrichingChannel(channel, DialogueTracing.tracingTags(cf, uriIndexForInstrumentation));
                 channel = cf.isConcurrencyLimitingEnabled()
                         ? new ChannelToEndpointChannel(endpoint -> {
+                            if (endpoint.tags().contains("dialogue-no-endpoint-limit")) {
```

Comment:
Open to suggestions on the name. I wanted to clearly namespace it to dialogue, rather than all clients, because this is very much an implementation detail of dialogue.

I don't expect this feature to be used very often, so the name may not be terribly important.

Reply:
I don't have a strong opinion. I might prefer something to make it clear that this is for concurrency limiting as opposed to request size, etc.

```suggestion
                            if (endpoint.tags().contains("dialogue-disable-endpoint-concurrency-limiting")) {
```

Reply:
I like it, thanks! Updated